app/views/static/buttons.html.haml

```diff
@@ -0,0 +1,45 @@
+.button-group
+  %button{:class     => "btn btn-primary btn-disabled",
+          :alt       => "Add",
```

Comment:
already commented on the other file, but adding here since this is the one actually used .. missing i18n for alt and title..